index.html

```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="styles.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter&family=Poppins&family=Roboto&display=swap"
+      rel="stylesheet"
+    />
+    <title>My Portfolio</title>
+  </head>
+  <body>
+    <nav>
+      <label class="mylogo">My Logo</label>
+      <ul class="four-menu">
+        <li><a href="#">Portfolio</a></li>
+        <li><a href="#">About</a></li>
+        <li><a href="#">Contact</a></li>
+        <li>
+          <a href="#"><img class="mail" src="images/mail.png" alt="Mail" /></a>
+        </li>
+      </ul>
+      <img class="toggle" src="images/hamburger.png" alt="toggle" />
+    </nav>
```

Comment:
well done by using nav tag but it's meant to wrap navigation menu you can wrap the desktop menu in a nav and the humberger link in another one also please make sure to wrap this section in a header tag  as this is good for semantic reasons and to make your code well organized and clean

Reply:
Do you mean like that -
<header>
      <nav>
        <label class="mylogo">My Logo</label>
        <ul class="four-menu">
          <li><a href="#">Portfolio</a></li>
          <li><a href="#">About</a></li>
          <li><a href="#">Contact</a></li>
          <li>
            <a href="#"><img class="mail" src="images/mail.png" alt="Mail" /></a>
          </li>
        </ul>
        <img class="toggle" src="images/hamburger.png" alt="toggle" />
      </nav>
    </header>